### PR TITLE
Added turn indicators

### PR DESCRIPTION
**Pull request by dcoollx**
Fixed EOL problem. now only shows proper changes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
 			<div class="undo" id="undo">
 				<p>UNDO</p>
 			</div>
+			<div id = "turnIndicator" >Reds turn</div>
```
Comment (ketanhwr) [inline index.html]: I think you need to add `style = "color:red"` here so that the turn is known for the first time also.

**Review by ketanhwr — CHANGES_REQUESTED** [review]
I think this is fine, take a look into my comment.

**Comment by ketanhwr** [comment]
@dcoollx recently, a pull request was merged that allowed more than 2 players. Therefore a few conflicts will be there and you will have to modify the logic to support more than 2 players as well.

You can either close this pull request, update your fork with the latest `ketanhwr:master` and make another pull request with the new logic, or you can rebase your pull request over my current master. I'll suggest the first since you will have a nicer commit history.

**Comment by dcoollx** [comment]
will create a new one with multiplayer logic per your request

**Pull request closed without merging**